client/src/pages/TagDirectoryPage/TagDirectoryPage.js

```diff
@@ -24,34 +29,82 @@ class TagDirectoryPage extends React.Component {
+            <div className="page-container">
+                <div className="tag-directory page static">
+                    <Helmet title="Tags and Materials | Monuments and Memorials"/>
+                    <Spinner show={fetchTagsPending}/>
+                    <div className="static-page-body">
+                        <div className="row">
+                            <div className="col-12">
+                                <div className="static-page-title">
+                                    <h1>Tag Directory</h1>
+                                </div>
+                                <div className="static-page-text">
+                                    <p>
+                                        This page provides a list of every tag and material in our system.
+                                        When deciding which tags or materials to add to your monument or memorial,
+                                        we suggest using existing ones first before resorting to creating new ones.
+                                    </p>
+                                </div>
+                            </div>
+                        </div>
+                        <div className="row">
+                            {this.renderAlphanumericFilter()}
+                            <div className="col-12 col-lg-6 mb-4 mb-lg-0">
+                                <div className="static-page-title">
+                                    <h2>Tags ({allTags.length})</h2>
+                                </div>
+                                <div className="static-page-text">
+                                    <Tags tags={allTags} selectable={false}/>
+                                </div>
+                            </div>
+                            <div className="col-12 col-lg-6">
+                                <div className="static-page-title">
+                                    <h2>Materials ({allMaterials.length})</h2>
+                                </div>
+                                <div className="static-page-text">
+                                    <Tags tags={allMaterials} selectable={false}/>
+                                </div>
+                            </div>
+                        </div>
                     </div>
                 </div>
+                <Footer/>
+            </div>
+        );
+    }
+
+    renderAlphanumericFilter() {
+        const { tags } = this.props;
+        const { alphanumericFilter } = this.state;
+        const characters = '012345679abcdefghijklmnopqrstuvwxyz'.split('')
+            .filter(character => {
+                return tags.find(tag => tag.name.toLowerCase().startsWith(character));
+            });
+        return (
+            <div className="alphanumeric-filter">
+                <span className={`character${alphanumericFilter === '' ? ' selected' : ''}`}>
+                    <a href="javascript:void();"
```

Comment:
What does this do?

Reply:
It makes the link all linky without actually linking to anything. If you don't have an href it just looks like regular text